src/AccessibilityHelperTrait.php

```diff
@@ -66,11 +68,12 @@ public function setReflectionClassInstance($instance, $class = null) {
  *
  * @param string $class
  * @return Object
+ * @throws Exception
```

Comment:
You just gave the explanation yourself: "Using PHP Storm the doc blocks not having a fully qualified namespace means that none of the code chasing or auto suggestion works".
So yeah all doc block lines need fully qualified namespace declarations for all classes used. That is totally independent if you use `use` statements or use the classes with `\`.